client/src/api/graduates.js

```diff
@@ -0,0 +1,33 @@
+export const graduates = () =>{
+	let grdats= {
+		"OBakir90":false,
+		"csfilipinyi":true,
+		"Buchraatkeh1984":true,
+        "FarhanaFKhan":false,
+        "juditlehoczki":false
+    };
+	return grdats;
+};
+
+
+export const graduateProfile = (name) =>{
+	let graduates= {
+        OBakir90:{
+            first_name:'Orhan',
+            last_name:'Bakir',
+            skills:['HTML', 'JavaScript', 'ReactJs', 'NodeJs']
+        },
+        csfilipinyi:{
+            first_name:'Cs',
+            last_name:'Flinpinyi',
+            skills:['JavaScript', 'ReactJs', 'NodeJs', 'Java']
+        },
+        Buchraatkeh1984:{
+            first_name:'Buchra',
+            last_name:'Atkeeh',
+            skills:['CSS', 'ReactJs', 'NodeJs', 'ReactNative']
+        }
+    }
+	return graduates[name];
+};
+
```

Comment:
I know this is temporary here until you get the real data from backend but I don't see the need for functions? You could just have two nested objects exported.
```javascript
export const graduates = {
        OBakir90: {
            first_name: 'Orhan',
            last_name: 'Bakir',
            skills: ['HTML', 'JavaScript', 'ReactJs', 'NodeJs']
        },
        csfilipinyi: {
            first_name: 'Cs',
            last_name: 'Flinpinyi',
            skills: ['JavaScript', 'ReactJs', 'NodeJs', 'Java']
        },
        Buchraatkeh1984: {
            first_name: 'Buchra',
            last_name: 'Atkeeh',
            skills: ['CSS', 'ReactJs', 'NodeJs', 'ReactNative']
        }
}
```
Then instead of calling `graduateProfile("OBakir90")` you can just do `graduates.OBakir90` or `graduates[username]`;